app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ProfilesController < PeopleController
+  before_action :user?
```

Comment:
Będziemy tutaj potrzebowali wywołania `user_requirede` analogicznego do `admin_required`. To wywołanie musi być zaimplementowane w kontrolerze `application_controller`.